src/plugins/maps_dashboards/public/model/ILayerConfig.ts

```diff
@@ -0,0 +1,9 @@
+
+//TODO: this is used for UI component development, remove or update it once layer model ready
```

Comment:
do we have a github issue or sprint task to address this todo item?

Reply:
Yes, just created a gihub issue to track this. https://github.com/opensearch-project/dashboards-maps/issues/56 Thanks!